Analysis/include/Particle.h

```diff
@@ -0,0 +1,38 @@
+/*! Definiton of Particle
+This file is part of https://github.com/hh-italian-group/h-tautau. */
+
+#pragma once
+
+#include <string>
+#include <set>
+#include "AnalysisTools/Core/include/Tools.h"
+
+namespace particles {
+namespace ParticleCode {
+
+    enum pdg {e= 11, mu =13, tau = 15 , b = 5, higgs = 25, nu_e = 12, nu_mu = 14, nu_tau = 16, gamma= 22, g= 21};
+    enum ParticleType {baryon, meson, lepton, diquark};
```

Comment:
- ParticleType should be defined outside ParticleCode namespace and defined as `enum class`
- Use ENUM_NAMES macro to define string representation of ParticleType (same as for the other enums)